bundle/config/target.go

```diff
@@ -8,6 +8,11 @@ import (
 
 type Mode string
 
+// We alias it here to override the JSON schema associated with a variable value
+// in a target override. This is because we allow for directly specifying the value
+// in addition to the variable.Variable struct format in a target override.
+type TargetVariable variable.Variable
+
```

Comment:
Could you move this into the `config/variable` package? That way the types for variables are colocated.

Reply:
done